Documentation/docs/learn/faq.md

```diff
@@ -0,0 +1,561 @@
+author = {Luis Ib\'{a}{\~n}ez and William Schroeder and Lydia Ng and Joshua Cates},
+title = {The {ITK} {S}oftware {G}uide},
+organization = {Kitware, Inc.},
+edition = {First},
+year = {2003},
+note = {ISBN 1-930934-10-6},
+url = {}
+}
+```
+
+```
+@manual{ITKSoftwareGuideSecondEdition,
+author = {Luis Ib\'{a}{\~n}ez and William Schroeder and Lydia Ng and Joshua Cates},
+title = {The {ITK} {S}oftware {G}uide},
+organization = {Kitware, Inc.},
+edition = {Second},
+year = {2005},
+note = {ISBN 1-930934-15-7},
+url = {}
+}
+```
+
+```
+@manual{ITKSoftwareGuideThirdEdition,
+author = {Hans J. Johnson and Matthew M .McCormick and Luis Ib\'{a}{\~n}ez and The Insight Software Consortium},
+title = {The {ITK} {S}oftware {G}uide},
+organization = {Kitware, Inc.},
+edition = {Third},
+year = {2013},
+note = {},
+url = {}
+}
+```
+
+- **A specific software version**
+
+See [https://zenodo.org/record/3592082](https://zenodo.org/record/3592082).
+
+If you have a publication that used ITK, please create a pull request to
+add it to [`ITKBibliography.bib`](https://github.com/InsightSoftwareConsortium/insightsoftwareconsortium.org/blob/master/static/citations-visualization/ITKBibliography.bib).
+
+If you want to include ITK in an acknowledgment section, a phrase
+similar to the following may be used:
+
+```
+"This work benefited from the use of the Insight Segmentation
+ and Registration Toolkit (ITK), an open source software
+ developed as an initiative of the U.S. National Library
+ of Medicine and available at www.itk.org."
+```
+
+and
+
+```
+"The multi-platform configuration tool CMake was used for
+ configuring ITK and facilitating its use from our project.
+ CMake was partially funded by the U.S. National Library of
+ Medicine as part of the Insight Toolkit project. CMake is an
+ open source system and it is freely available at www.cmake.org."
+```
+
+### Is ITK FDA-Approved?
+
+Given the fact that ITK is a software toolkit, it cannot be the subject
+of FDA approval as a medical device. We have discussed this topic in
+several occasions and received advice from FDA representatives, that can
+be summarized as follows:
+
+```
+"ITK is to be considered as an off-the-shelf (OTS) product that
+ is used for supporting a higher level medical application/product.
+ The developer of such application/product will be responsible for
+ performing the validation processes described in FDA published
+ guidelines for the development of software-related medical devices."
+```
+
+For mode details see the page [FDA Guidelines for Software Developement](https://itk.org/Wiki/FDA_Guidelines_for_Software_Development).
+
+## Using ITK
+
+### How do I combine ITK and VTK in my application?
+
+A full tutorial on how to combine ITK and VTK is available at [Getting Started with ITK and VTK](http://www.itk.org/CourseWare/Training/GettingStarted-II.pdf).
+
+You will need the auxiliary classes available at the [ITKApps](https://github.com/InsightSoftwareConsortium/ITKApps)
```

Comment:
As per https://github.com/InsightSoftwareConsortium/ITK/pull/4313#discussion_r1390272297, mentions to `ITKApps` should be removed. @thewtex ?